Assets/Src/Scripts/SeedCalc/CuttingBoard.cs

```diff
@@ -0,0 +1,82 @@
+// Copyright 2021 The Aha001 Team.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Collections;
+using UnityEngine;
+
+namespace SeedCalc {
+  // The cutting board to show visualizable numbers and reference objects.
```

Comment:
Why name it as `cutting board`?

Reply:
It's a common term or metaphor preferred by UX designers - something like a show board on the desk. Joho and the UX intern use this word in their original design files.